fe/fe-core/src/test/java/com/starrocks/sql/analyzer/AnalyzeLateralTest.java

```diff
@@ -40,6 +44,25 @@ public void testUnnest() {
                 "Table function must be used with lateral join");
 
         analyzeFail("select * from t0,unnest(bitmap_to_array(bitmap_union(to_bitmap(v1))))",
-                "UNNEST clause cannot contain aggregations");
+                "Table Function clause cannot contain aggregations");
+    }
+
+    @Test
+    public void testVarArgs() {
+        analyzeSuccess("select * from tarray, unnest(v3) as t(unnest_1)");
+        analyzeSuccess("select unnest_1 from tarray, unnest(v3) as t(unnest_1)");
+        analyzeSuccess("select t.unnest_1 from tarray, unnest(v3) as t(unnest_1)");
+        analyzeFail("select t.unnest_2 from tarray, unnest(v3) as t(unnest_1)");
+
+        analyzeSuccess("select t.unnest_1 from tarray, unnest(v3, ['a', 'b', 'c']) as t(unnest_1, unnest_2)");
```

Comment:
did you test unnest(null, null)?

Reply:
Yes, I have tested. The current performance is consistent with PostgreSQL.
When null literal appears in unnset, such as
SELECT * FROM demo,UNNEST(null,null)
SELECT * FROM demo,UNNEST(ARRAY[1,2,3],null)
will report the error "db error: ERROR: function pg_catalog.unnest(unknown) is not unique HINT: Could not choose a best candidate function. You might need to add explicit type casts."